src/alliteratedActors.js

```diff
@@ -21,8 +21,12 @@
  *
  * @return {string}
  */
-function alliteratedActors(names) {
-  // write code here
-}
+// eslint-disable-next-line max-len
+const alliteratedActors = names => {
```

Comment:
якщо в стрілочній функції відриваєш `{}`, то бажано параметр обгортати дужками
```suggestion
const alliteratedActors = (names) => {
```